src/server/handlers/openapi/mojaloop/consentRequests/{ID}.ts

```diff
@@ -20,14 +20,29 @@
 
  * Google
  - Steven Wijaya <stevenwjy@google.com>
+ - Abhimanyu Kapur <abhi.kapur09@gmail.com>
  --------------
  ******/
 
 import { Request, ResponseToolkit } from '@hapi/hapi'
 import { Handler, Context } from 'openapi-backend'
 import { logger } from '~/shared/logger'
+import { consentRepository } from '~/repositories/consent'
+import { ConsentStatus } from '~/models/consent'
 
-export const put: Handler = async (context: Context, request: Request, h: ResponseToolkit) => {
+export const put: Handler = async (
+  context: Context,
+  request: Request,
+  h: ResponseToolkit
+) => {
   logger.logRequest(context, request, h)
```

Comment:
I think we can remove the logging from master for this and other handlers

Reply:
makes sense

Reply:
actually why not leave it in?

Reply:
wouldn't it make sense for a demo server to keep a record of transactions, linking etc.

Reply:
not sure about this, but personally I don't think its a good idea to just log everything in a function like this (which will be called so many times)... I usually prefer to just record some critical parts or if there is any error... how about other mojaloop services? do they log every requests?